server/api/flavors.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const db = require('APP/db');
+const Flavors = db.model('flavors');
```

Comment:
just for consistency, can we name this const Flavor (same as User, Seller, etc.)?